build-tools-internal/src/main/java/org/elasticsearch/gradle/internal/NoticeTask.java

```diff
@@ -35,22 +40,29 @@
 public class NoticeTask extends DefaultTask {
 
     @InputFile
-    private File inputFile = getProject().getRootProject().file("NOTICE.txt");
+    private File inputFile;
+
     @OutputFile
-    private File outputFile = new File(getProject().getBuildDir(), "notices/" + getName() + "/NOTICE.txt");
+    private File outputFile;
```

Comment:
Can we put a linebreak in here so this is more readable?

Reply:
done